cypress/common/if-exists.js

```diff
@@ -1,15 +1,25 @@
 /**
```

Comment:
suggestion (non-blocking): If you wanted to you could change the name of the file to match the function name.